internal/command/register_commands.go

```diff
@@ -0,0 +1,7 @@
+package command
+
+import "haracho-go/internal/command/ping_pong"
+
+func RegisterCommands() {
+	ping_pong.Init()
+}
```

Comment:
```suggestion
import (
	"haracho-go/internal/client"
	"haracho-go/internal/command/pingpong"
)

func RegisterCommands(collection *client.CommandCollection) {
	pingpong.Build(collection)
}
```
`CommandCollection` を受け取って書き加えるだけにするべき